Assets/Scripts/Gameplay/Player/PlayerAttack.cs

```diff
@@ -8,21 +8,33 @@ public class PlayerAttack : MonoBehaviour
     public float Timer = 0.0f;
     public float attackSpeed = 2.0f;
     public float WaitTime = 2.0f;
-    // Start is called before the first frame update
+
+    private PlayerStats playerStats;
+
+
+    public void Start()
+    {
+        playerStats = GetComponent<PlayerStats>(); 
```

Comment:
Should Add a RequiresComponent to the class if you are going to assume it has it here, otherwise it needs to be wrapped in a != null check.